### PR TITLE
LWIMIQ July 18th

### PR DESCRIPTION
**Pull request by chrisarcand**
@JPrause

### CODE REVIEW

**Comment by JPrause** [comment]
@syncrou @imtayadeway @juliancheal can you each review this week's blog entry.

File: site/_posts/2017-07-18-kicking-off-the-summer-edition.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+### Miscellaneous
+
+A PORT environment variable ([manageiq#15536](https://github.com/ManageIQ/manageiq/pull/15536)) and some
+leading blank lines ([manageiq#15515](https://github.com/ManageIQ/manageiq/pull/15515)) made their
+way to the chopping blank, too. Yay deleting stuff!
+
+## Wrapping up
+
+As always, this is just a small sampling of all the great pull requests merged
+to ManageIQ these past couple weeks. Check out the ManageIQ organization on
```
Comment (imtayadeway) [inline site/_posts/2017-07-18-kicking-off-the-summer-edition.md]: Might be cool to link to this for lazy people like me? 😁

File: site/_posts/2017-07-18-kicking-off-the-summer-edition.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+Github to view the progress being made across all of our repositories, and
```
Comment (imtayadeway) [inline site/_posts/2017-07-18-kicking-off-the-summer-edition.md]: Think this should be s/Github/GitHub/

File: site/_posts/2017-07-18-kicking-off-the-summer-edition.md (hunk 1 above)
Comment (chrisarcand) [inline site/_posts/2017-07-18-kicking-off-the-summer-edition.md]: Oops forgot to go back and do that, thanks

**Comment by JPrause** [comment]
LGTM! Thanks again @chrisarcand 
@cybette for your review and merge.

File: site/_posts/2017-07-18-kicking-off-the-summer-edition.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+solstice (for the northern hemisphere) already occurred on June 20th. And wait,
+that's not even the start of meteorological summer which was actually on June
+1st. Also shouldn't it be Last Week**s** In ManageIQ, then?
+
+No matter. Let's not sweat the details in this hot weather and just get to the good stuff.
+
+
+## Featured
+
+[60 pull requests][manageiq PRs merged] were merged in to ManageIQ core over
```
Comment (cybette) [inline site/_posts/2017-07-18-kicking-off-the-summer-edition.md]: perhaps s/in to/into/

File: site/_posts/2017-07-18-kicking-off-the-summer-edition.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+longer find yourself staring at it, since it's now fixed! You've got better
+things to do.
+
+### Fix VNC console connection to Windows VMs and Ctrl-Alt-Del button ([manageiq-ui-classic#1672](https://github.com/ManageIQ/manageiq-ui-classic/pull/1672))
+Similarly, you've got better things to do than stare at non-responsive buttons. Great news! These aforementioned buttons are fixed.
+
+## New
+
+### Bundler groups ([manageiq#15459](https://github.com/ManageIQ/manageiq/pull/15459))
+Ruby dependencies in ManageIQ's Gemfile are now bucketed in to [Bundler groups](http://bundler.io/v1.15/groups.html), which
```
Comment (cybette) [inline site/_posts/2017-07-18-kicking-off-the-summer-edition.md]: Sorry for nitpicking... I think it should be "into" here as well

File: site/_posts/2017-07-18-kicking-off-the-summer-edition.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+## Deleted
+
+### Save inventory container: remove target option ([manageiq#15182](https://github.com/ManageIQ/manageiq/pull/15182))
+The `target` option in the save inventory container was removed as it was unused, greatly simplifying partial graph refreshes in the future.
+
+### Miscellaneous
+
+A PORT environment variable ([manageiq#15536](https://github.com/ManageIQ/manageiq/pull/15536)) and some
+leading blank lines ([manageiq#15515](https://github.com/ManageIQ/manageiq/pull/15515)) made their
+way to the chopping blank, too. Yay deleting stuff!
```
Comment (cybette) [inline site/_posts/2017-07-18-kicking-off-the-summer-edition.md]: Should it be chopping block? or is that another way of saying it?

Comment (JPrause) [inline site/_posts/2017-07-18-kicking-off-the-summer-edition.md]: @chrisarcand I think this might be the last edit needed before we can merge.

Comment (chrisarcand) [inline site/_posts/2017-07-18-kicking-off-the-summer-edition.md]: HAH, yeah that should be block.

**Comment by chrisarcand** [comment]
@cybette @JPrause Fixed! :shipit:

**Pull request merged**